docs/conf.py

```diff
@@ -38,28 +34,3 @@ class intergov:
 pygments_style = 'sphinx'
 html_theme = 'alabaster'  #'default'
 html_static_path = ['_static']
-htmlhelp_basename = 'intergov_ledgerdoc'
-
-latex_elements = {
-    'papersize': 'a4paper',
-}
-latex_documents = [
-    ('index', 'intergov_ledger.tex',
-     u'Intergov Ledger Documentation',
-     u'Chris Gough', 'manual'),
-]
-
```

Comment:
why kill latexpdf? is it not used by RTD?

Reply:
Oh, does it. Just thought we didn't use it. I haven't finished reading (I haven't really started, not a high priority) but I think that readthedocs can't do apidoc generation for us anyway, so we would have to push autogenerated .rst to get it the rest of the way, may as well just build it as we want and tell readthedocs where to look for it. That's my current thinking...

Reply:
OK, that would be fine.

I think maybe you can make slightly evil hooks in conf.py to trick it RTD into doing interesting things... but having a pre-commit generation step is probably more hygenic.